_config.example.yml

```diff
@@ -0,0 +1,14 @@
+canvas-token: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
+gitlab-host: https://git-classes.mst.edu/
+namespace: 2016-Fall-CS1001
+roster:
```

Comment:
Let's leave a comment here, something like `# we recommend using one of the import commands to automatically populate the roster`